decomposed/extensions/MDDE_LDM/Profile/Model/Methods/mdde_InitializeExportExtension.xml

```diff
@@ -32,57 +51,80 @@ Function GTLOutputHasErrors(GTLResult)
+         success2 = applyTemplateOnObject (objModel, ent, attributeTemplateName, attributeExtendedAttributeName)
+         
+         If success1 = False Or success2 = False Then
+            processPackage = False
+         End If
       End If   
    Next
    
    &#39;Loop over EntityAttributes
    Dim entAttr
    For Each entAttr in objPackage.EntityAttributes
       If Not entAttr.IsShortCut Then
-         applyTemplateOnObject objModel, entAttr, &quot;mddex_EntityAttribute_Export_Template&quot;, &quot;mddex_EntityAttribute_Export_Content&quot;
-         applyTemplateOnObject objModel, entAttr, attributeTemplateName, attributeExtendedAttributeName            
+         success1 = applyTemplateOnObject (objModel, entAttr, &quot;mddex_EntityAttribute_Export_Template&quot;, &quot;mddex_EntityAttribute_Export_Content&quot;)
+         success2 = applyTemplateOnObject (objModel, entAttr, attributeTemplateName, attributeExtendedAttributeName)
+         
+         If success1 = False Or success2 = False Then
+            processPackage = False
+         End If
       End If   
    Next
    
    &#39;Loop over Mappings
    Dim map
    For Each map in objPackage.Mappings
        If Not map.IsShortCut Then
-         applyTemplateOnObject objModel, map, &quot;mddex_BaseClassifierMapping_Export_Template&quot;, &quot;mddex_BaseClassifierMapping_Export_Content&quot;
-         applyTemplateOnObject objModel, map, attributeTemplateName, attributeExtendedAttributeName            
+         success1 = applyTemplateOnObject (objModel, map, &quot;mddex_BaseClassifierMapping_Export_Template&quot;, &quot;mddex_BaseClassifierMapping_Export_Content&quot;)
+         success2 = applyTemplateOnObject (objModel, map, attributeTemplateName, attributeExtendedAttributeName)
+         
+         If success1 = False Or success2 = False Then
+            processPackage = False
+         End If
       End If   
    Next
    
    &#39;Loop over AttributeMappings
    Dim attrMap
    For Each attrMap in objPackage.ModelEntityAttributeMappings
       If Not attrMap.IsShortCut Then
-         applyTemplateOnObject objModel, attrMap, &quot;mddex_BaseStructuralFeatureMapping_Export_Template&quot;, &quot;mddex_BaseStructuralFeatureMapping_Export_Content&quot;         
+         success1 = applyTemplateOnObject (objModel, attrMap, &quot;mddex_BaseStructuralFeatureMapping_Export_Template&quot;, &quot;mddex_BaseStructuralFeatureMapping_Export_Content&quot;)
+         
+         If success1 = False Then
+            processPackage = False
+         End If
       End If   
    Next
    
    &#39;Loop over packages
    Dim childPackage
    For Each childPackage in objPackage.Packages
-      processPackage objModel, childPackage, attributeTemplateName, attributeExtendedAttributeName
+      success1 = processPackage (objModel, childPackage, attributeTemplateName, attributeExtendedAttributeName)
```

Comment:
Why don't you need success2 part here, like in the other loops?

Reply:
Because the packages are handled differently then other metaclases since it is a recursive invocation of the method